nvm.sh

```diff
@@ -116,12 +116,17 @@ nvm_get_latest() {
 }
 
 nvm_download() {
-  local CURL_COMPRESSED_FLAG
   if nvm_has "curl"; then
+    local CURL_OPTIONS=()
+
+    if [ -n "$NVM_AUTH_HEADER" ]; then
+      CURL_OPTIONS+=(--header "$NVM_AUTH_HEADER")
+    fi
+
     if nvm_curl_use_compression; then
-      CURL_COMPRESSED_FLAG="--compressed"
+      CURL_OPTIONS+=(--compressed)
```

Comment:
never seen this syntax before; does this work in POSIX?

Reply:
Yeah, the array syntax actually worked like a charm... until I realized its not POSIX compliant 🫤